CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog - v3
 
+## [v3.6.4] (July 19 2023)
+### Feat:
+* Created a separate package.json for CommonJS (cjs) module during build time. This package.json is located under dist/cjs directory. (#687)
```

Comment:
Can we use the present tense for consistency?

Reply:
Yeah updated it https://github.com/sendbird/sendbird-uikit-react/pull/689/commits/612d30e9b25ce77d76162cfef6a832ad30d284f2